platform-includes/llm-rules-platform/_default.mdx

```diff
@@ -1,21 +1,81 @@
 <Expandable title="AI Rules for Cursor or Windsurf" copy={true}>
 
-Sentry provides a set of rules you can use to help your LLM use Sentry correctly. Copy this file and add it to your projects rules configuration. 
+Sentry provides a set of rules you can use to help your LLM use Sentry correctly. Copy this file and add it to your projects rules configuration.
 
 ````markdown {filename:rules.md}
-
 These examples should be used as guidance when configuring Sentry functionlaity within a project.
 
 # Error / Exception Tracking
 
 - Use `Sentry.captureException(error)` to capture an exception and log the error in Sentry.
 - Use this in try catch blocks or areas where exceptions are expected
 
-# Logs 
+# Tracing Examples
+
+- Spans should be created for meaningful actions within an applications like button clicks, API calls, and function calls
+- Use the `Sentry.startSpan` function to create a span
+- Child spans can exist within a parent span
+
+## Custom Span instrumentation in component actions
+
+- Name custom spans with meaningful names and operations.
+- Attach attributes based on relevant information and metrics from the request
+
+```javascript
+function TestComponent() {
+  const handleTestButtonClick = () => {
+    // Create a transaction/span to measure performance
+    Sentry.startSpan(
+      {
+        op: "ui.click",
+        name: "Test Button Click",
+      },
+      (span) => {
+        const value = "some config";
+        const metric = "some metric";
+
+        // Metrics can be added to the span
+        span.setAttribute("config", value);
+        span.setAttribute("metric", metric);
+
+        doSomething();
+      }
+    );
+  };
+
+  return (
+    <button type="button" onClick={handleTestButtonClick}>
+      Test Sentry
+    </button>
+  );
+}
+```
+
+## Custom span instrumentation in API calls
+
+- Name custom spans with meaningful names and operations.
+- Attach attributes based on relevant information and metrics from the request
+
+```javascript
+async function fetchUserData(userId) {
+  return Sentry.startSpan(
+    {
+      op: "http.client",
+      name: `GET /api/users/${userId}`,
+    },
+    async () => {
+      const response = await fetch(`/api/users/${userId}`);
+      const data = await response.json();
+      return data;
+    }
+  );
+}
+```
+
+# Logs
 
-- Where logs are used, ensure Sentry is imported using `import * as Sentry from "@sentry/browser"` 
+- Where logs are used, ensure Sentry is imported using `import * as Sentry from "@sentry/browser"`
 - Reference the logger using `const { logger } = Sentry`
-- Sentry initialization needs to be updated to include the `logger` integration.
```

Comment:
Lets still include that `enableLogs: true` should be set within the `_experiments` configuration block to reinforce the snippet below. Calling it a logger integration was wrong in my case.